Content.Shared/_RMC14/Xenonids/Construction/Nest/XenoNestSystem.cs

```diff
@@ -187,6 +192,7 @@ private void OnNestSurfaceDoAfter(Entity<XenoNestSurfaceComponent> ent, ref Xeno
         _transform.SetLocalRotation(victim, direction.ToAngle());
 
         _standing.Stand(victim, force: true);
+        _parasite.RefreshIncubationMultipliers(victim);
```

Comment:
Can you move this to a <XenoNestedComponent, ComponentAdded> subscription instead?